### ISSUE TITLE
some suggestions

### ISSUE THREAD

**Issue by joeljosephjin** [sudharsan13296/Awesome-Meta-Learning#5]
**Interesting Papers:**

Tasks, stability, architecture, and compute: Training more effective learned optimizers, and using them to train themselves
Metz et al. (2020)

M2SGD: Learning to Learn Important Weights
Kuo et al. (2020)

Meta-Learning in Neural Networks: A Survey
Hospedales et al., (2020)

Hyper-Meta Reinforcement Learning with Sparse Reward
Hua et al., (2020)

Meta-Curvature
Park et al., (2020)

Learning to Learn via Self-Critique
Antoniou et al., (2019)

La-MAML: Look-ahead Meta Learning for Continual Learning
Gupta et al. (2020)

**Lectures:**

https://www.youtube.com/watch?v=CRHKgOYXVe8

**Useful libraries:**

Learn2Learn - https://github.com/learnables/learn2learn
Higher - https://github.com/facebookresearch/higher
pytorch-meta - https://github.com/tristandeleu/pytorch-meta

cheers!

**Comment by YinDeGuan** [comment]
nice bro , nice everybody :)
